content/_includes/site/hire-us.njk

```diff
@@ -0,0 +1,27 @@
+{% import 'embed.macros.njk' as embed %}
+
+{%- set title = '<span class="title-action">Hire Us to Build</span> Custom Web Apps <span class="title-description">That stand the test of time</span>' -%}
+
+<aside data-layout="cta" aria-labelledby="hire-cta-title" class="hire-us-cta" data-ccs-colors="dark">
+  {% set cta_content %}
+    <h2 id="hire-cta-title" data-header='stacked hire-cta'>{{ title | mdInline | safe }}</h2>
```

Comment:
`title` isn't markdown, it's just HTML in this case -- is there a reason to `{% set %}` it instead of having it inline here?

Reply:
removed this in https://github.com/oddbird/oddleventy/pull/808/commits/b0426139237a6137623da2ce46314a44fe0de654